packages/core/src/app/shipping/hooks/useMultishippingConsignmentItems.ts

```diff
@@ -52,8 +61,7 @@ function mapConsignmentsItems(
     lineItems.physicalItems.forEach((item) => {
         unassignedItemsMap.set(item.id.toString(), { ...item, type: LineItemType.Physical });
         itemHashMap.set(item.id.toString(), generateItemHash(item));
-    }
-    );
+    });
```

Comment:
Shall we filter the items here (or before this) so that the `itemHash` and all does not need to be generated for picklist items.